[2023-11-22T05:00:23.722+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rand_user.is_api_available manual__2023-11-22T05:00:19.467094+00:00 [queued]>
[2023-11-22T05:00:23.751+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rand_user.is_api_available manual__2023-11-22T05:00:19.467094+00:00 [queued]>
[2023-11-22T05:00:23.752+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-11-22T05:00:23.752+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 3
[2023-11-22T05:00:23.753+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-11-22T05:00:23.789+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-11-22 05:00:19.467094+00:00
[2023-11-22T05:00:23.801+0000] {standard_task_runner.py:55} INFO - Started process 3946 to run task
[2023-11-22T05:00:23.809+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'rand_user', 'is_api_available', 'manual__2023-11-22T05:00:19.467094+00:00', '--job-id', '589', '--raw', '--subdir', 'DAGS_FOLDER/predictions.py', '--cfg-path', '/tmp/tmp358k34gz']
[2023-11-22T05:00:23.814+0000] {standard_task_runner.py:83} INFO - Job 589: Subtask is_api_available
[2023-11-22T05:00:23.955+0000] {task_command.py:376} INFO - Running <TaskInstance: rand_user.is_api_available manual__2023-11-22T05:00:19.467094+00:00 [running]> on host 91f7624ceef9
[2023-11-22T05:00:24.143+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Gaurav
AIRFLOW_CTX_DAG_ID=rand_user
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-11-22T05:00:19.467094+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-22T05:00:19.467094+00:00
[2023-11-22T05:00:24.145+0000] {http.py:120} INFO - Poking: api/
[2023-11-22T05:00:24.163+0000] {base.py:71} INFO - Using connection ID 'rand_user' for task execution.
[2023-11-22T05:00:24.167+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-11-22T05:00:24.609+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-11-22T05:00:24.626+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=rand_user, task_id=is_api_available, execution_date=20231122T050019, start_date=20231122T050023, end_date=20231122T050024
[2023-11-22T05:00:24.704+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-11-22T05:00:24.776+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
